modules/ROOT/pages/appliance/installation.adoc

```diff
@@ -60,9 +59,9 @@ from you DNS configuration.
 Don’t Forget the *IP Address* and the *Administrator Password*. You will
 need them to use the Appliance.
 
-[[start-the-appliance]]
-Start the Appliance
--------------------
+[[the-configuration-wizard]]
+Configuration wizard
+--------------------
```

Comment:
Nice spot, but if you use [native AsciiDoc header formatting](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#document-header), you no longer need to worry about this.

Reply:
Sure would be nice to have a workshoppy introduction to all these new features and rules :)